### ISSUE TITLE
`tb.toggleLayer` does not work for one object if another glb model is also loaded

### ISSUE THREAD

**Issue by John2397** [jscastro76/threebox#140]
I have two objects on the map, a geometry object and a glb object.

`tb.toggleLayer`(or `map.setLayoutProperty` as they are the same ) does not work if both of the objects are loaded into the map.

If I remove the glb layer, `tb.toggleLayer`  works correctly.

[This fiddle ](https://jsfiddle.net/c50ofyks/) describes the problem.  It it supposed to hide the `box_layer` but it doesn't.
If I remove the line `map.addLayer(glb_model_layer)`, then `box_layer` is correctly hidden

`statueofliberty.glb` and `threebox` need to be added on fiddle

**Comment by jscastro76** [comment]
Hi John,
That's because you are creating 2 layers, both of them with a call to `tb.update()` on the render method.
Check the [example I added on multilayer use](https://github.com/jscastro76/threebox/blob/master/examples/16-multilayer.html) that will show you how to work with multiple layers at the same time, not only for explicit toggle but also for zoom-based toggle.
![image](https://user-images.githubusercontent.com/38470852/101652580-b12f9100-3a3e-11eb-8707-5afe78eade17.png)

Basically, you need to define `multiLayer: true` in your threebox initialization, and remove the `tb.update` from the render methods in your layers. Threebox will do the rest of the magic for you.

**Comment by jscastro76** [comment]
Hi @John2397 
Did you resolved your issue with the layers visibility?? May I close this one?

**Comment by George2397** [comment]
Sorry didn't find the time to test it yet, but you can close it for now. I'm sure this is the problem!
Totally missed this example. Thank you!

**Comment by John2397** [comment]
Sorry didn't find the time to test it yet, but you can close it for now. I'm sure this is the problem!
Totally missed this example. Thank you!
